api/V1/Recording.js

```diff
@@ -196,12 +212,7 @@ module.exports = (app, baseUrl) => {
             { "$or": [{ public: true }, { GroupId: { "$in": userGroupIds } }] }
           ],
         },
-        order: [
-            // Sort by recordingDatetime but handle the case of the
-            // timestamp being missing and fallback to sorting by id.
-            [sequelize.fn("COALESCE", sequelize.col('recordingDateTime'), '1970-01-01'), "DESC"],
-            ["id", "DESC"],
-        ],
+        order: order,
```

Comment:
Is there any danger in taking input straight from the API call and using it here? Does sequelize do all the right escaping to prevent SQL injection attacks?